docs/htdocs/info/docs/api/variation/variation_schema.html

```diff
@@ -313,7 +313,7 @@ <h2><a class="sql_schema_link" href="#header_13">Variation tables</a></h2>
   <div class="sql_schema_group_bullet" style="background-color:#FF0000"></div>
   <h2>Attributes tables</h2>
 </div>
-<p class="sql_schema_group_header_desc">These tables define the variation attributes data.</p><div align="center"><a href="diagrams/Variation.attributes_tables.png"><img src="diagrams/Variation.attributes_tables.png" width="216px" /></a></div><div class="sql_schema_table">
+<p class="sql_schema_group_header_desc">These tables define the variation attributes data.</p><div align="center"><a href=".//Variation.attributes_tables.png"><img src=".//Variation.attributes_tables.png" width="216px" /></a></div><div class="sql_schema_table">
```

Comment:
Are these supposed to be `.//` instead of `diagrams/`? The images are still in the `diagrams/` folder.